modules/3d/include/opencv2/3d/depth.hpp

```diff
@@ -0,0 +1,1264 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#ifndef __OPENCV_RGBD_DEPTH_HPP__
+#define __OPENCV_RGBD_DEPTH_HPP__
+
+#include <opencv2/core.hpp>
+#include <limits>
+
+namespace cv
+{
+//! @addtogroup rgbd
+//! @{
+
+  /** Checks if the value is a valid depth. For CV_16U or CV_16S, the convention is to be invalid if it is
+   * a limit. For a float/double, we just check if it is a NaN
+   * @param depth the depth to check for validity
+   */
+  CV_EXPORTS
+  inline bool
```

Comment:
avoid indentation in namespaces

Reply:
fixed indentation across the file & in some src files